package org.CalculatorProject.challenge.level3.exception;

public class CustomExceptionHandler {
    public static void handleException(Exception e) {
        System.out.println(e.getMessage());
    }
}
